tests/bench/CMakeLists.txt

```diff
@@ -20,12 +20,12 @@ include_directories(
   ${CMAKE_BINARY_DIR}
 )
 include_directories(SYSTEM
-  ${SQLITE3_INCLUDE_DIR}
+  ${SQLite3_INCLUDE_DIRS}
```

Comment:
include_directories() should not longer be needed because the below target_link_libraries() now uses the SQLite::SQLite3 target name, which declares INTERFACE_INCLUDE_DIRECTORIES per your above changes (cf https://schneide.blog/2016/04/08/modern-cmake-with-target_link_libraries/)

Reply:
Good point! I'll make the changes.